packages/contracts-ops/contracts/RotateUpdater.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+pragma abicoder v2;
+
+import {Config} from "./Config.sol";
+import {CallBatch} from "./CallBatch.sol";
+
+import {UpdaterManager} from "@nomad-xyz/contracts-core/contracts/UpdaterManager.sol";
+import {Home} from "@nomad-xyz/contracts-core/contracts/Home.sol";
+import {Replica} from "@nomad-xyz/contracts-core/contracts/Replica.sol";
+
+import "forge-std/Script.sol";
+
+contract RotateUpdater is Script, Config, CallBatch {
+    function run(
+        string calldata configFile,
+        string calldata localDomain,
+        string calldata output
+    ) public {
+        __Config_initialize(configFile);
+        __CallBatch_initialize(localDomain, output);
+
+        string[] memory connections = connections(localDomain);
+        address newUpdater = updater(localDomain);
+
+        Home home = home(localDomain);
+        UpdaterManager updaterManager = updaterManager(localDomain);
+
+        if (newUpdater != home.updater()) {
+            push(
+                address(home),
+                abi.encodeWithSelector(home.setUpdater.selector, newUpdater)
```

Comment:
is this correct? my reading of Home is that this will fail, but I documented this as necessary. cc @anna-carroll

Reply:
calling directly would fail. need to call through UpdaterManager, as you've changed the code to do :)